packages/components/src/duotone-picker/duotone-picker.js

```diff
@@ -80,13 +98,16 @@ function DuotonePicker( {
 			}
 		>
 			{ ! disableCustomColors && ! disableCustomDuotone && (
-				<CustomDuotoneBar value={ value } onChange={ onChange } />
+				<CustomDuotoneBar
+					value={ isUnset ? undefined : value }
+					onChange={ onChange }
+				/>
 			) }
 			{ ! disableCustomDuotone && (
 				<ColorListPicker
 					labels={ [ __( 'Shadows' ), __( 'Highlights' ) ] }
 					colors={ colorPalette }
-					value={ value }
+					value={ isUnset ? undefined : value }
```

Comment:
We do this 3 times. Should we make it a variable?

Reply:
It's only used in two places—ColorListPicker and CustomDuotoneBar—and I can't think of a good name for what to call it that would make the code any more readable. Do you have a suggestion?